### PR TITLE
[scoped-custom-element-registry] Adds `shadowRoot.createElementNS`

### LINKED ISSUE #563: [scoped-custom-element-registry] createElementNS is unsupported
### Description

With the scoped-custom-element-registry polyfill installed. When calling `createElementNS()` via a shadowRoot an error is thrown.

### Example
```html
<div id="app"></div>
```

```js
const appDiv = document.getElementById('app');
const shadowRoot = appDiv.attachShadow({ mode: 'open' });
const div = shadowRoot.createElementNS('http://www.w3.org/1999/xhtml', 'div');
```

### Steps to reproduce

1. Install the scoped-custom-element-registry polyfill
2. Attach a shadow root to an element
3. Call `shadowRoot.createElementNS()`

#### Expected behavior

No error is thrown as the polyfill should support createElementNS as it is included in the [spec](https://github.com/WICG/webcomponents/blob/gh-pages/proposals/Scoped-Custom-Element-Registries.md).

#### Actual behavior

Error is thrown `TypeError: .createElementNS is not a function`

### Version

@webcomponents/scoped-custom-element-registry@0.0.9

### Browsers affected

- [X] Chrome
- [X] Firefox
- [X] Edge
- [X] Safari
- [x] IE 11

### PR DESCRIPTION
**Pull request by sorvell**
Fixes #563

### CODE REVIEW

File: packages/scoped-custom-element-registry/test/ShadowRoot.test.html.js [hunk 1]
```diff
@@ -292,6 +292,34 @@ describe('ShadowRoot', () => {
       });
     });
 
+    describe('createElementNS', () => {
```
Comment (justinfagnani) [inline packages/scoped-custom-element-registry/test/ShadowRoot.test.html.js:295]: We also need tests in the "with custom registry" suite, yes?

File: packages/scoped-custom-element-registry/CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- shadowRoot.createElementNS
```
Comment (justinfagnani) [inline packages/scoped-custom-element-registry/CHANGELOG.md:14]:
```suggestion
- Added support for ShadowRoot.prototype.createElementNS()
```

File: packages/scoped-custom-element-registry/CHANGELOG.md [hunk 3]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
 - registry in ShadowRootInit; matches current proposal but customElements
```
Comment (justinfagnani) [inline packages/scoped-custom-element-registry/CHANGELOG.md:16]: Just some cleanup suggestions from previous PRs:

```suggestion
- Added the `registry` property to ShadowRootInit to match current proposal. `customElements`
```

File: packages/scoped-custom-element-registry/test/ShadowRoot.test.html.js (hunk 1 above)
Comment (sorvell) [reply packages/scoped-custom-element-registry/test/ShadowRoot.test.html.js:340]: Oops, done.

**Review by justinfagnani — APPROVED** [review]

**Pull request merged**